scipy/stats/_continuous_distns.py

```diff
@@ -4089,6 +4089,60 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_normal_gen(rv_continuous):
+    """A skew-normal random variable
+    
+    %(before_notes)s
+    
+    Notes
+    -----
+    `skewnormal` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to a normal.
+    rvs implements the method of http://azzalini.stat.unipd.it/SN/faq-r.html
+    %(after_notes)s
+    
+    %(example)s
+    
+    >>>import matplotlib.pyplot as plt
+    >>>x = np.linspace(-5,5,100)
+    >>>plt.hist(stats.skewnormal.rvs(a=4, size=1e6), bins=50, normed=True, color='red') 
+    >>>plt.hist(stats.skewnormal.rvs(a=-4, size=1e6), bins=50, normed=True, color='green') 
+    >>>plt.hist(stats.skewnormal.rvs(a=0, size=1e6), bins=50, normed=True, color='blue') 
+    >>>plt.plot(x, stats.skewnormal.pdf(x,a=4), 'r')
+    >>>plt.plot(x, stats.skewnormal.pdf(x,a=-4), 'g')
+    >>>plt.plot(x, stats.skewnormal.pdf(x,a=0), 'b')
+    >>>plt.show()
+    """
+
+    def _argcheck(self, a):
+        return True
```

Comment:
This reflects the fact that `a` is allowed to be negative? Needs to be tested then.